client/gutenberg/extensions/publicize/settings-button.jsx

```diff
@@ -49,14 +49,11 @@ class PublicizeSettingsButton extends Component {
 	render() {
 		return (
 			<div className="jetpack-publicize-add-connection-container">
-				<span
-					className="jetpack-publicize-add-icon dashicons-plus-alt"
-				>
-				</span>
 				<a
 					onClick={ this.settingsClick }
 					tabIndex="0"
 				>
+					<span className="jetpack-publicize-add-icon dashicons-plus-alt" />
```

Comment:
Want to prettier this? That would handle self-closing 😁

Reply:
Will do it for the whole Publicize extension in a follow-up PR 👍

Reply:
Done in #28428